### DISCUSSION TITLE
Specifying the `-W` flag to install Lerna-Lite packages with `npm`

### DISCUSSION THREAD

**Discussion by mvila** [ghiscoding/lerna-lite#613] [Q&A] [answered]
To my knowledge, the `-W` flag is not supported by `npm`, and I think it is a bit confusing to see it in the README.

For example:

```
$ npm i @lerna-lite/publish -D -W
```

**Comment by ghiscoding** [comment]
ah I see, it looks like I'm half wrong, it's `-w` (lowercase) 🤦🏻‍♂️ 
https://docs.npmjs.com/cli/v9/using-npm/workspaces?v=true#adding-dependencies-to-a-workspace

Would you like to contribute a PR? If you do, please review all readme (including the ones in each command)
Thanks

**Comment by mvila** [comment]
@ghiscoding Sure. I can create a PR, but even in lowercase (`-w`), I don't see the point of adding this flag, which is usually used to specify a workspace. Since Lerna-Lite packages are typically installed at the root, we don't need to select a workspace.

I think the confusion comes from `yarn`, which has a `-W` (uppercase) flag explicitly specifying that a package should be installed at the root.

**Reply by ghiscoding** [reply, marked as answer]
yeah I used to work with yarn in the past and now use pnpm in monorepo structure and the `-w` is required for these 2, perhaps it's not required for npm I didn't know. Feel free to remove it in a PR if you want. It was supposed to help but yes it might be more confusing than anything.
